docs/Velox.md

```diff
@@ -129,18 +129,32 @@ cd /path_to_gluten/cpp
 cd /path_to_gluten
 mvn clean package -Pbackends-velox -Pspark-3.2 -Pfull-scala-compiler -DskipTests -Dcheckstyle.skip
 ```
-Gluten HDFS support requires an extra environment variable "VELOX_HDFS" to indicate the Hdfs URI. e.g. VELOX_HDFS="host:port". If the env variable is missing, Gluten will try to connect with hdfs://localhost:9000
+Gluten HDFS support requires Hdfs URI, you can define this config in three ways:
 
-This env should be exported in both Spark driver and worker.
-e.g., in Spark local mode:
+1. spark-defaults.xml
```

Comment:
I think you meant for `spark-defaults.conf`

Reply:
already fix this.